### DISCUSSION TITLE
LG plugin no longer working with Apple Home

### DISCUSSION THREAD

**Discussion by eranhuman** [nVuln/homebridge-lg-thinq#268] [Q&A]
I recently uninstalled my LG plugin (for my washer and dryer) to setup a new bridge and reinstall. It all works fine in Hoobs and I can see the appliances but when adding the new bridge to my apple home the applicances are not importing themselves. would love any advice. (was working fine prior to me messing with it).

**Comment by nVuln** [comment]
I'm not sure what happened in Hoobs, you can try: remove homebridge out of Home app first, then reset all cached accessories, reset homebridge to grant new pairing code, then try re-add it again

**Reply by eranhuman** [reply]
yup tried that - no change. the bridge installs but doesn't bring in the accessories - not sure why :(
